java-agent/src/jvmMain/kotlin/com/epam/drill/agent/instrument/reactor/ReactorTransformer.kt

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2020 - 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.drill.agent.instrument.reactor
```

Comment:
**option 1**
Yes, we can group this class and all classes referenced in `reactorTransformers` to the 
`com.epam.drill.agent.instrument.reactor.transformers` package & dir - for clarity

**option 2**
**EDIT**: or, what we can do is group classes by "instrumentation subject". Following the structure: 

- ReactorTransformer (a nice top-level class - good for a top-down view when reading the code first time)
- Subscriber
  transformer
  interceptor
- Flux
  transfomer
  interceptor

etc